### PR TITLE
hooks: add hook for easyocr

### PR DESCRIPTION
**Pull request by Jmennius**
I hope what I've done is not too controversial :)

### CODE REVIEW

**Comment by bwoodsend** [comment]
Ahh phew, you had me expecting `collect_all()` used in a for loop plus some hard-coded brew paths and perhaps some cuda dlls.

We do already have a mechanism for hooks with parameters. See [here](https://pyinstaller.org/en/stable/hooks-config.html) for the about and [here](https://pyinstaller.org/en/stable/hooks-config.html#adding-an-option-to-the-hook) for adding options. Since you're the first to use it in this repo, you'll need to create a new *Supported hooks options* markdown file to put the documentation in. We'll later need to add a link to it in https://pyinstaller.org/en/stable/hooks-config.html#supported-hooks-and-options.

**Comment by Jmennius** [comment]
> Ahh phew, you had me expecting `collect_all()` used in a for loop plus some hard-coded brew paths and perhaps some cuda dlls.
> 
> We do already have a mechanism for hooks with parameters. See [here](https://pyinstaller.org/en/stable/hooks-config.html) for the about and [here](https://pyinstaller.org/en/stable/hooks-config.html#adding-an-option-to-the-hook) for adding options. Since you're the first to use it in this repo, you'll need to create a new _Supported hooks options_ markdown file to put the documentation in. We'll later need to add a link to it in https://pyinstaller.org/en/stable/hooks-config.html#supported-hooks-and-options.

Done.
Feel free to rephrase/move or otherwise fix the doc. I've kept the hook-options doc ReST and it's in the root of the repo... 🤷🏼‍♂️

**Comment by Jmennius** [comment]
I think there is another part that is quite crucial for this package - it downloads NN models (at runtime).
So I probably have to add it. Not sure when - maybe another PR, but definitely not this week :)

**Comment by bwoodsend** [comment]
> I think there is another part that is quite crucial for this package - it downloads NN models (at runtime).

That depends on where it puts them and what it needs to do it. If it writes to somewhere sensible like `~/.cache` and doesn't require any dependencies that PyInstaller isn't already able to detect then we should be able to leave it alone. If it downloads into its own `site-packages` location then it'll error out under PyInstaller when someone installs to someone owned by root.

File: hooks-config.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+===========================
+Supported hooks and options
+===========================
+
+This section lists hooks that implement support for configuration
+options in `pyinstaller-hooks-contrib` repository.
```
Comment (bwoodsend) [inline hooks-config.rst:6]:
```suggestion
This section lists hooks that are configurable in `pyinstaller-hooks-contrib`.
```

File: hooks-config.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Please refer to `Hook Configuration Options <https://pyinstaller.org/en/stable/hooks-config.html>`_
```
Comment (bwoodsend) [inline hooks-config.rst:8]:
```suggestion
See also `Hook Configuration Options <https://pyinstaller.org/en/stable/hooks-config.html>`_
```

**Comment by Jmennius** [comment]
@bwoodsend I've squashed your suggestions!

**Review by bwoodsend — APPROVED** [review]
Looks good to me. I'd be tempted to ask for a test too but given that easyocr depends on pytorch+cuda, I think I'd rather not go there.

**Review by rokm — APPROVED** [review]

**Comment by bwoodsend** [comment]
Nice one!

**Pull request merged**